lib/parsers/dfxp.js

```diff
@@ -1,58 +0,0 @@
-const { Parser } = require('xml2js');
```

Comment:
I know you already explained it to me, but can you comment here on why it makes sense to remove the dfxp parser so we have a record of this decision?

Reply:
TTML is the newer version of DFXP. Since the getExtension function cannot distinguish between these formats, it will return a '.ttml' result for both file types.
After running a diff comparison on our .ttml.js and .dfxp.js parser files we can see that the logic is identical in both cases(see the result below).
The only differences are:
- the `const Joi = require('joi-browser'); `import is on the 3rd rather than 2nd line
- the name of the functions(ttml (...) vs dfxp(...))
- the if statement on line 43 is multiline in one file and a single line in other.

As the function will be never called it is safe to delete the dfxp.js file. This will avoid code duplication, and will still provide support for .dfxp conversion by using .ttml.js.

`...$ diff lib/parsers/dfxp.js lib/parsers/ttml.js 
2d1
< const Joi = require('joi-browser');
3a3
> const Joi = require('joi-browser');
34c34
< function dfxp(subtitleText, options) {
---
> function ttml(subtitleText, options) {
43,45c43
<     if (err) {
<       status.invalidEntries.push(err);
<     }
---
>     if (err) status.invalidEntries.push(err);
58c56,57
< module.exports = dfxp;
---
> 
> module.exports = ttml;`